index.d.ts

```diff
@@ -1,21 +1,21 @@
-declare module 'apollo-datasource-mongodb' {
-  import { DataSource } from 'apollo-datasource'
+declare module 'apollo-mongo-datasource' {
```

Comment:
Isnt this supposed to be apollo-datasource-mongodb itself since we are making changes to the same package?

Reply:
You're right, I was originally going to create a separate package because I wasn't sure this repo was still being maintained. Just fixed the module name, should be good now

Reply:
IMO, I think it would be better to maintain this as a separate package, since I dont see this getting merged anytime soon.
This is something my team has also been dependent on.